inspirehep/utils/cv_latex.py

```diff
@@ -128,6 +126,8 @@ def _format_output_row(self, field, value):
                 field, value)
         elif field == 'url':
             out += u' %\href{{{1}}}{{HEP entry}}.\n'.format(field, value)
+        elif field == 'citation_count':
+            out += u' %{1}'.format(field, value)
```

Comment:
Well in that case there is no need actually as we can have u' %{0}'.format(value)
It was mainly used for Bibtex where we had sth like       pages          = "079",
where pages where field and 079 value. I think we can remove it actually.